tests/test-meta.php

```diff
@@ -325,28 +317,21 @@ public function register_script_hooks_provider() {
 	}
 
 	public function test_register_scripts_checks_for_existing_scripts() {
```

Comment:
We can safely drop some of the tests that were playing the "oh, if someone else has already registered Select2, we can use their version" — we should be the only ones registering/enqueueing Sticky Tax scripts + styles.

Reply:
I removed all the Select2 stuff, just kept the name of the test function itself.

Reply:
fwiw the test itself is still valid, even if it's an edge case now that we have our own CSS and JS